widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageView.kt

```diff
@@ -65,9 +69,13 @@ class MessageView(
     private var attachmentButtonClickListener: OnClickListener? = null
     private var sendMessageButtonClickListener: ((String) -> Unit)? = null
     private var onMessageTextChangedListener: ((String) -> Unit)? = null
+    private var onRemoveAttachmentListener: ((FileAttachment) -> Unit)? = null
```

Comment:
NIT: Does it make sense to declare `typealias Listener<T> = (T) -> Unit` and use it instead of `((FileAttachment) -> Unit)? = null`, `((String) -> Unit)? = null` like this `Listener<FileAttachment>? = null` and `Listener<String>? = null`

Reply:
Good idea.

Reply:
I will refactor it in the next secure conversation task.